requirements.txt

```diff
@@ -1,14 +1,53 @@
+appdirs==1.4.4
```

Comment:
We definitely don't need to add all these dependencies.

Reply:
These are dependencies of django-allauth package, we need these for django-allauth to work